Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# 1.4.2 Adaptation

## Changelog:
- Adapted the mod to 1.4.2.
- The mod no longer freezes on the loading screen of the map (when you click the new game and select a bookmark).


# How to test:
Before testing, copy-paste `Warcraft-Guardians-of-Azeroth-2.mod` from `Crusader Kings III\mod\Warcraft-Guardians-of-Azeroth-2` into `Crusader Kings III\mod`.

Confirm the freezing fix.
